datadog_checks_downloader/tests/test_downloader.py

```diff
@@ -250,10 +224,10 @@ def test_local_wheels_signer_signature_leaf_error(distribution_name, distributio
             http_url,
         ]
 
-        with pytest.raises(NoWorkingMirrorError) as exc:
+        with pytest.raises(UnsignedMetadataError) as exc:
             _do_run_downloader(argv)
 
-    assert "BadSignatureError('wheels-signer-" in str(exc)
+    assert "wheels-signer-a was signed by 0/1 keys" in str(exc)
```

Comment:
Don't you want to propagate any original message included in `UnsignedMetadataError` instead of assuming what must have happened?

Reply:
I'm not sure what you mean. `str(exc)` equals `<ExceptionInfo UnsignedMetadataError('wheels-signer-a was signed by 0/1 keys') tblen=11>`, I don't see any more specific info about what happened in there.

Reply:
You can use `match` in `pytest.raises` to make sure expected string is available ([docs](https://docs.pytest.org/en/7.1.x/how-to/assert.html#assertions-about-expected-exceptions)). Something like:

```python
with pytest.raises(UnsignedMetadataError, match="^wheels-signer-a was signed by 0/\d+ keys$"):
    _do_run_downloader(argv)
```

Reply:
Could we please quickly fix this now?